src/pages/BooksPage.js

```diff
@@ -17,18 +17,33 @@ const BooksPage = () => {
     dispatch(removeBookItem(id));
   };
 
+  const btnStyle = {
+    color: '#fff',
+    backgroundColor: '#0290ff',
+    padding: '3px 5px',
+    cursor: 'pointer',
+    border: 'none',
+    borderRadius: '3px',
+  };
+
   return (
-    <div className="books-section">
+    <section className="books-section">
       <div className="books-container">
         {books.map((book) => (
-          <Book key={book.item_id} book={book} onClick={() => handleRemoveBook(book.item_id)} />
+          <Book
+            key={book.item_id}
+            book={book}
+            btnStyle={btnStyle}
+            onClick={() => handleRemoveBook(book.item_id)}
+          />
         ))}
       </div>
```

Comment:
Good job working on the project requirements however most of your font colors are not matching the template design it would be nice to carefully insect the template design for the correct font colors, font, font sizes.

This is your design:



![Screenshot 2023-07-22 at 2 44 13 PM](https://github.com/over-geek/BookStore/assets/79506105/0e1be6e4-bd3c-4f04-a519-91ad123397e6)


Please follow this design:


![Screenshot 2023-07-22 at 2 44 44 PM](https://github.com/over-geek/BookStore/assets/79506105/d0ff6318-e03c-4a83-8e16-9c5404c57613)

It is important to pay attention to detail following design guidelines is important.